main/lz4/APKBUILD

```diff
@@ -1,16 +1,16 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=lz4
-pkgver=1.8.3
-pkgrel=2
+pkgver=1.9.0
+pkgrel=0
 pkgdesc="LZ4 is lossless compression algorithm with fast decoder @ multiple GB/s per core."
 url="https://github.com/lz4/lz4"
 arch="all"
 license="BSD-2-Clause GPL-2.0-only"
 checkdepends="diffutils"
+options="!check" # disabled due to incompatibility with drone https://github.com/lz4/lz4/issues/675
```

Comment:
If the issue is specific with drone we shouldn't disable the tests

Reply:
The tests pass in my development environment & [under `travis`](https://travis-ci.org/alpinelinux/aports/builds/521363338?utm_source=github_status&utm_medium=notification).

If this `PR` can be pulled with `drone` failing I will change it back

Reply:
fixed in https://github.com/lz4/lz4/pull/686 & applied in `fix-drone-tests.patch`